pkg/cli/start.go

```diff
@@ -1121,6 +1121,20 @@ func setupAndInitializeLoggingAndProfiling(
 		)
 	}
 
+	// The new multi-region abstractions require that nodes be labeled in "regions"
+	// (and optionally, "zones").  To push users in that direction, we warn them here
+	// if they've provided a --locality value which doesn't include a "region" tier.
+	if len(serverCfg.Locality.Tiers) > 0 {
+		if _, containsRegion := serverCfg.Locality.Find("region"); !containsRegion {
```

Comment:
do we have to check whether `region` is the first tier, or does all out logic allow that?